.ci/atime/tests.R

```diff
@@ -129,9 +129,10 @@ test.list <- list(
   pkg.edit.fun = pkg.edit.fun,     
   N = 10^seq(1, 7),
   setup = quote({ 
-    DT = replicate(N, 1, simplify = FALSE)
+    L <- replicate(N, 1, simplify = FALSE)
+    data.table:::setattr(L, "class", NULL)    
```

Comment:
setattr should be moved inside expr, please see https://github.com/Rdatatable/data.table/pull/5427#issuecomment-2073368617

Reply:
Oops yes you're right